File: 3656-minimum-number-of-operations-to-make-elements-in-array-distinct/minimum-number-of-operations-to-make-elements-in-array-distinct.cpp
class Solution {
public:
    int minimumOperations(vector<int>& nums) {
        bitset<101>vis;
        for(int i = nums.size()-1 ; i>=0;i--)
            if(vis[nums[i]])return i/3+1;
            else vis[nums[i]] =1;
        return 0;
    }
};

File: 3656-minimum-number-of-operations-to-make-elements-in-array-distinct/minimum-number-of-operations-to-make-elements-in-array-distinct.py
class Solution:
    def minimumOperations(self, nums: List[int]) -> int:
        n = len(nums)
        frq = [-1]*101
        for i in range(n-1,-1,-1):
            if frq[nums[i]] != -1:
                return (i+3)//3
            frq[nums[i]] = i
        return 0

       

